### ISSUE TITLE
manual/javaGuide/main/i18n/JavaI18N.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#958]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/i18n/JavaI18N.md
- diffs between [2.2.0/manual/javaGuide/main/i18n/JavaI18N.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/javaGuide/main/i18n/JavaI18N.md) and [2.3.5/manual/javaGuide/main/i18n/JavaI18N.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/i18n/JavaI18N.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/main/i18n/JavaI18N.md 2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/javaGuide/main/i18n/JavaI18N.md    2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # Externalising messages and internationalization

 ## Specifying languages supported by your application
@@ -37,7 +38,7 @@
 '''
 ## Formatting messages

-Messages can be formatted using the `java.text.MessageFormat` library. For example, if you have defined a message like this:
+Messages are formatted using the `java.text.MessageFormat` library. For example, if you have defined a message like this:

 '''
 files.summary=The disk {1} contains {0} file(s).
@@ -49,6 +50,20 @@
 Messages.get("files.summary", d.files.length, d.name)
 '''

+## Notes on apostrophes
+
+Since Messages uses `java.text.MessageFormat`, please be aware that single quotes are used as a meta-character for escaping parameter substitutions.
+
+For example, if you have the following messages defined:
+
+@[single-apostrophe](code/javaguide/i18n/messages)
+@[parameter-escaping](code/javaguide/i18n/messages)
+
+you should expect the following results:
+
+@[single-apostrophe](code/javaguide/i18n/JavaI18N.java)
+@[parameter-escaping](code/javaguide/i18n/JavaI18N.java)
+
 ## Retrieving supported languages from an HTTP request

 You can retrieve a specific HTTP request’s supported languages:

```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1119
